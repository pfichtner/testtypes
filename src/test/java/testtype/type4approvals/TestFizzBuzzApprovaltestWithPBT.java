package testtype.type4approvals;

import static net.jqwik.api.ShrinkingMode.OFF;
import static org.approvaltests.combinations.CombinationApprovals.verifyAllCombinations;
import static testtype.FizzBuzz.fizzBuzz;

import java.util.List;

import net.jqwik.api.Arbitraries;
import net.jqwik.api.ForAll;
import net.jqwik.api.Property;
import net.jqwik.api.Provide;
import net.jqwik.api.arbitraries.ListArbitrary;
import net.jqwik.api.constraints.IntRange;

/**
 * An approval test based on jqwik. Please note that to get this work a modified
 * version of {@link org.approvaltests.namer.AttributeStackSelector} was copied
 * to this project.
 */
class TestFizzBuzzApprovaltestWithPBT {

	@Property(tries = 1, seed = "-1787866974758305853", shrinking = OFF)
	void approve1To100(@ForAll("listsWithIntegersBetween1And100") List<Integer> numbers) {
		verifyAllCombinations(i -> fizzBuzz(i), numbers.toArray(new Integer[numbers.size()]));
	}

	@Provide
	ListArbitrary<Integer> listsWithIntegersBetween1And100() {
		// pick 250 random integer values
		return Arbitraries.integers().greaterOrEqual(1).list().uniqueElements().ofSize(250);
	}

}
