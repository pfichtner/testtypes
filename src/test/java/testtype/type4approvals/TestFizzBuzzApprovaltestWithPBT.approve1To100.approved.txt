[4141] => 4141 
[312] => Fizz 
[42285] => FizzBuzz 
[10] => Buzz 
[67] => 67 
[151338] => Fizz 
[2147483647] => 2147483647 
[47467] => 47467 
[1974644938] => 1974644938 
[258658] => 258658 
[14342] => 14342 
[33] => Fizz 
[104] => 104 
[3454921] => 3454921 
[855] => FizzBuzz 
[1122] => Fizz 
[944] => 944 
[11909] => 11909 
[413764] => 413764 
[2] => 2 
[973873] => 973873 
[10884] => Fizz 
[124] => 124 
[23] => 23 
[7750] => Buzz 
[42] => Fizz 
[4190] => Buzz 
[2064400527] => Fizz 
[11402] => 11402 
[801] => Fizz 
[339] => Fizz 
[1] => 1 
[897963] => Fizz 
[64] => 64 
[545588] => 545588 
[143251] => 143251 
[432] => Fizz 
[1198] => 1198 
[2695766] => 2695766 
[113] => 113 
[2147483646] => Fizz 
[609405] => FizzBuzz 
[15] => FizzBuzz 
[4926550] => Buzz 
[1723986] => Fizz 
[52364] => 52364 
[22] => 22 
[8] => 8 
[19] => 19 
[12] => Fizz 
[3928] => 3928 
[4800096] => Fizz 
[155792] => 155792 
[121834] => 121834 
[323766578] => 323766578 
[152310] => FizzBuzz 
[3712961] => 3712961 
[233] => 233 
[892] => 892 
[80621] => 80621 
[2632905] => FizzBuzz 
[66] => Fizz 
[7] => 7 
[11690] => Buzz 
[151] => 151 
[758621] => 758621 
[1996291356] => Fizz 
[105952] => 105952 
[2199830] => Buzz 
[780861] => Fizz 
[828460511] => 828460511 
[1693] => 1693 
[58] => 58 
[4904] => 4904 
[341] => 341 
[2064] => Fizz 
[104375] => Buzz 
[482] => 482 
[1018] => 1018 
[8276] => 8276 
[398225] => Buzz 
[5097269] => 5097269 
[1785] => FizzBuzz 
[65971] => 65971 
[5299] => 5299 
[33289] => 33289 
[139652] => 139652 
[170] => Buzz 
[1027709] => 1027709 
[128010] => FizzBuzz 
[106426] => 106426 
[1026] => Fizz 
[534406] => 534406 
[374107845] => FizzBuzz 
[3] => Fizz 
[198] => Fizz 
[155570] => Buzz 
[410593] => 410593 
[6] => Fizz 
[26894] => 26894 
[244] => 244 
[365998] => 365998 
[134389] => 134389 
[313] => 313 
[4772741] => 4772741 
[5946] => Fizz 
[1545537071] => 1545537071 
[126409] => 126409 
[514] => 514 
[3354] => Fizz 
[3039] => Fizz 
[844373617] => 844373617 
[4732650] => FizzBuzz 
[837586] => 837586 
[1395] => FizzBuzz 
[2327554] => 2327554 
[2143] => 2143 
[2947] => 2947 
[163] => 163 
[765235] => Buzz 
[114] => Fizz 
[669] => Fizz 
[250410639] => Fizz 
[69] => Fizz 
[1325] => Buzz 
[1975203237] => Fizz 
[20667] => Fizz 
[5692] => 5692 
[2673119] => 2673119 
[138963] => Fizz 
[6142] => 6142 
[2299] => 2299 
[54624] => Fizz 
[1090616587] => 1090616587 
[36] => Fizz 
[1475301411] => Fizz 
[278] => 278 
[655] => Buzz 
[76670] => Buzz 
[4] => 4 
[141801] => Fizz 
[85] => Buzz 
[1163] => 1163 
[174946] => 174946 
[7249] => 7249 
[897386] => 897386 
[52774] => 52774 
[54] => Fizz 
[473913] => Fizz 
[9571] => 9571 
[279] => Fizz 
[1587221875] => Buzz 
[528] => Fizz 
[13924] => 13924 
[657508] => 657508 
[81832] => 81832 
[18046] => 18046 
[32833] => 32833 
[32] => 32 
[505748] => 505748 
[158365] => Buzz 
[139130] => Buzz 
[446] => 446 
[84050] => Buzz 
[1001160] => FizzBuzz 
[5599] => 5599 
[4185917] => 4185917 
[4648583] => 4648583 
[3034873] => 3034873 
[161] => 161 
[224] => 224 
[115] => Buzz 
[1704180294] => Fizz 
[4269] => Fizz 
[25] => Buzz 
[1471] => 1471 
[2088983439] => Fizz 
[1361784] => Fizz 
[6941] => 6941 
[274] => 274 
[29244] => Fizz 
[557167] => 557167 
[13] => 13 
[39811] => 39811 
[97] => 97 
[4143512] => 4143512 
[47337] => Fizz 
[1327287911] => 1327287911 
[5197046] => 5197046 
[468] => Fizz 
[5367191] => 5367191 
[16023] => Fizz 
[6946] => 6946 
[132] => Fizz 
[1189508943] => Fizz 
[5439727] => 5439727 
[229] => 229 
[200302] => 200302 
[50] => Buzz 
[441253] => 441253 
[293218] => 293218 
[671] => 671 
[28405] => Buzz 
[46433] => 46433 
[7485] => FizzBuzz 
[397] => 397 
[483070] => Buzz 
[83502] => Fizz 
[20] => Buzz 
[719945] => Buzz 
[1429010198] => 1429010198 
[4838] => 4838 
[676] => 676 
[72133] => 72133 
[1211] => 1211 
[1064] => 1064 
[120] => FizzBuzz 
[6213] => Fizz 
[697336] => 697336 
[142074] => Fizz 
[72588] => Fizz 
[37] => 37 
[1556125] => Buzz 
[421825] => Buzz 
[407] => 407 
[73439] => 73439 
[284463] => Fizz 
[1110622735] => Buzz 
[5027492] => 5027492 
[17481] => Fizz 
[49] => 49 
[207] => Fizz 
[153864966] => Fizz 
[255] => FizzBuzz 
[210] => FizzBuzz 
[768] => Fizz 
[45] => FizzBuzz 
[889318] => 889318 
[43] => 43 
[4544] => 4544 
[50676] => Fizz 
[3406848] => Fizz 
[112] => 112 
[14511] => Fizz 
[9633] => Fizz 
[11] => 11 
[100573] => 100573 
[72264] => Fizz 
[2118940068] => Fizz 
[943] => 943 
[1969771951] => 1969771951 
[1292] => 1292 
[1302135636] => Fizz 
[21] => Fizz 
[121] => 121 
[147] => Fizz 
[5093687] => 5093687 
[223] => 223 
[78601] => 78601 
[1675] => Buzz 
[410] => Buzz 
[411] => Fizz 
[9] => Fizz 
[9304] => 9304 
[107] => 107 
[28436] => 28436 
[665006] => 665006 
[5000379] => Fizz 
[32741] => 32741 
[178] => 178 
[87] => Fizz 
[59514] => Fizz 
[484] => 484 
[874] => 874 
[1888] => 1888 
[2008] => 2008 
[452] => 452 
[1857] => Fizz 
[5511543] => Fizz 
[538970] => Buzz 
[6517] => 6517 
[5621] => 5621 
[177] => Fizz 
[1996591] => 1996591 
[60] => FizzBuzz 
[2931] => Fizz 
[96624] => Fizz 
[168] => Fizz 
[55652] => 55652 
[23664] => Fizz 
[701] => 701 
[321] => Fizz 
[104036] => 104036 
[872585] => Buzz 
[800087] => 800087 
[14417] => 14417 
[372] => Fizz 
[1903264413] => Fizz 
[17328] => Fizz 
[238657] => 238657 
[211820433] => Fizz 
[1777] => 1777 
[93992] => 93992 
[5] => Buzz 
[5593280] => Buzz 
[215800610] => Buzz 
[45800] => Buzz 
[13876] => 13876 
[91] => 91 
[6183] => Fizz 
[17750] => Buzz 
[55] => Buzz 
[905] => Buzz 
[47982] => Fizz 
[7146] => Fizz 
[252] => Fizz 
[947] => 947 
[131347] => 131347 
[316195] => Buzz 
[45520] => Buzz 
[251550] => FizzBuzz 
[140] => Buzz 
[262070] => Buzz 
[31051] => 31051 
[5709542] => 5709542 
[853022] => 853022 
[45811] => 45811 
[2946954] => Fizz 
[3590] => Buzz 
[3495] => FizzBuzz 
[19716] => Fizz 
[12811] => 12811 
[2272127] => 2272127 
[21028] => 21028 
[4051] => 4051 
[55591] => 55591 
[149962] => 149962 
[85634] => 85634 
[3564524] => 3564524 
[1064017] => 1064017 
[163348] => 163348 
[5205534] => Fizz 
[524120717] => 524120717 
[1261464213] => Fizz 
[901178] => 901178 
[13028] => 13028 
[2014660] => Buzz 
[2634729] => Fizz 
[1075133226] => Fizz 
[4318784] => 4318784 
[59] => 59 
[80] => Buzz 
[9519] => Fizz 
[1215] => FizzBuzz 
[774] => Fizz 
[197] => 197 
[17129] => 17129 
[66668] => 66668 
[108] => Fizz 
[762] => Fizz 
[50653] => 50653 
[169] => 169 
[4275] => FizzBuzz 
[41880] => FizzBuzz 
[14] => 14 
[1346104504] => 1346104504 
[129] => Fizz 
[14694] => Fizz 
[3604641] => Fizz 
[259] => 259 
[48971] => 48971 
[821] => 821 
[214] => 214 
[579] => Fizz 
[626] => 626 
[2849] => 2849 
[11121] => Fizz 
[46017] => Fizz 
[280062] => Fizz 
[1051] => 1051 
[9477] => Fizz 
[4701] => Fizz 
[3946] => 3946 
[6979] => 6979 
[15118] => 15118 
[468380254] => 468380254 
[849144] => Fizz 
[5871] => Fizz 
[159] => Fizz 
[28] => 28 
[819] => Fizz 
[67295152] => 67295152 
[53017] => 53017 
[16] => 16 
[599777] => 599777 
[72] => Fizz 
[2096896] => 2096896 
[3094] => 3094 
[15695] => Buzz 
[2786] => 2786 
[216] => Fizz 
[439734] => Fizz 
[497] => 497 
[175] => Buzz 
[4424717] => 4424717 
[199] => 199 
[380] => Buzz 
[513814] => 513814 
[653] => 653 
[1003718787] => Fizz 
[4735374] => Fizz 
[700053] => Fizz 
[734] => 734 
[1143840402] => Fizz 
[2117475] => FizzBuzz 
[1781] => 1781 
[5044510] => Buzz 
[44] => 44 
[5936] => 5936 
[2120509077] => Fizz 
[4383] => Fizz 
[8000] => Buzz 
[314443] => 314443 
[37020] => FizzBuzz 
[47130] => FizzBuzz 
[461968] => 461968 
[129996] => Fizz 
[154043] => 154043 
[1198388354] => 1198388354 
[189] => Fizz 
[489095] => Buzz 
[2677340] => Buzz 
[72520] => Buzz 
[2476] => 2476 
[31283] => 31283 
[3074731] => 3074731 
[41379] => Fizz 
[145] => Buzz 
[56] => 56 
[9016] => 9016 
[235428621] => Fizz 
[814065862] => 814065862 
[209] => 209 
[953] => 953 
[10201] => 10201 
[16248] => Fizz 
[5601] => Fizz 
[3741428] => 3741428 
[548600] => Buzz 
[523097] => 523097 
[38] => 38 
[8812] => 8812 
[179] => 179 
[3431840] => Buzz 
[2076586476] => Fizz 
[93581] => 93581 
[4666278] => Fizz 
[69710] => Buzz 
[57] => Fizz 
[71] => 71 
[24] => Fizz 
[125836] => 125836 
[63] => Fizz 
[31054] => 31054 
[1186] => 1186 
[132601] => 132601 
[993175] => Buzz 
[1750] => Buzz 
[31301] => 31301 
[3042786] => Fizz 
[3300] => FizzBuzz 
[52] => 52 
[65] => Buzz 
[1380692530] => Buzz 
[1844472127] => 1844472127 
[967438] => 967438 
[5419] => 5419 
[13073] => 13073 
[693613] => 693613 
[723] => Fizz 
[330] => FizzBuzz 
[2071] => 2071 
[67144] => 67144 
[662] => 662 
[580031] => 580031 
[1914] => Fizz 
[76] => 76 
[139] => 139 
[6806] => 6806 
[297] => Fizz 
[1827] => Fizz 
[309] => Fizz 
[10685] => Buzz 
[156271] => 156271 
[17] => 17 
[5010] => FizzBuzz 
[115938] => Fizz 
[153] => Fizz 
[140925] => FizzBuzz 
[61487] => 61487 
[766] => 766 
[654519] => Fizz 
[28229] => 28229 
[35546787] => Fizz 
[148613] => 148613 
[2652] => Fizz 
[3200] => Buzz 
[3709474] => 3709474 
[9752] => 9752 
[5908] => 5908 
[2800250] => Buzz 
[1342198] => 1342198 
[422733] => Fizz 
[3141488] => 3141488 
[6130] => Buzz 
[103] => 103 
[129477] => Fizz 
[884618435] => Buzz 
[156] => Fizz 
[1621] => 1621 
[3107301] => Fizz 
[165452] => 165452 
[13904] => 13904 
[501] => Fizz 
[5141] => 5141 
[16908] => Fizz 
[263504] => 263504 
[1756] => 1756 
[698876] => 698876 
[53035] => Buzz 
[47057] => 47057 
[1323] => Fizz 
[1945333370] => Buzz 
[6619] => 6619 
[611] => 611 
[31286] => 31286 
[445] => Buzz 
[433564] => 433564 
[554] => 554 
[137] => 137 
[19318] => 19318 
[1944] => Fizz 
[202] => 202 
[28847] => 28847 
[1710522271] => 1710522271 
[18] => Fizz 
[1450523] => 1450523 
[84551] => 84551 
[147118] => 147118 
[28688] => 28688 
[93] => Fizz 
[14569] => 14569 
[1360284859] => 1360284859 
[483642] => Fizz 
[22791] => Fizz 
[1414] => 1414 
[1932] => Fizz 
[12724] => 12724 
[1138735787] => 1138735787 
[1067114864] => 1067114864 
[3065652] => Fizz 
[18069] => Fizz 
[7384] => 7384 
[9753] => Fizz 
[591803] => 591803 
[335] => Buzz 
[30] => FizzBuzz 
[555] => FizzBuzz 
[341468] => 341468 
[14692] => 14692 
[4657927] => 4657927 
[17973] => Fizz 
[13063] => 13063 
[64605] => FizzBuzz 
[9255] => FizzBuzz 
[642] => Fizz 
[1742] => 1742 
[658638] => Fizz 
[4395] => FizzBuzz 
[2298] => Fizz 
[118723] => 118723 
[347005768] => 347005768 
[112227] => Fizz 
[1526048547] => Fizz 
[793] => 793 
[333] => Fizz 
[1660447898] => 1660447898 
[631] => 631 
[150051] => Fizz 
[386] => 386 
[40660] => Buzz 
[99172] => 99172 
[5904] => Fizz 
[34899] => Fizz 
[153433] => 153433 
[94956] => Fizz 
[469241848] => 469241848 
[95651627] => 95651627 
[982766238] => Fizz 
[13177] => 13177 
[1287] => Fizz 
[8371] => 8371 
[69753] => Fizz 
[394052] => 394052 
[342873] => Fizz 
[5219] => 5219 
[1858507] => 1858507 
[547642111] => 547642111 
[152] => 152 
[1905923140] => Buzz 
[196515974] => 196515974 
[1959436] => 1959436 
[11373] => Fizz 
[54852] => Fizz 
[776] => 776 
[5195] => Buzz 
[34] => 34 
[2895893] => 2895893 
[93020] => Buzz 
[51344] => 51344 
[165] => FizzBuzz 
[3715] => Buzz 
[165894] => Fizz 
[21600] => FizzBuzz 
[11150] => Buzz 
[1037] => 1037 
[2712914] => 2712914 
[193] => 193 
[734907] => Fizz 
[846679648] => 846679648 
[260] => Buzz 
[6003639] => Fizz 
[2894] => 2894 
[75799] => 75799 
[51] => Fizz 
[624593] => 624593 
[147707] => 147707 
[164] => 164 
[36883] => 36883 
[355] => Buzz 
[132815] => Buzz 
[5361898] => 5361898 
[50598] => Fizz 
[953931] => Fizz 
[1253648] => 1253648 
[184] => 184 
[819780] => FizzBuzz 
[1935212] => 1935212 
[5704] => 5704 
[374] => 374 
[128067] => Fizz 
[41] => 41 
[258073] => 258073 
[2944] => 2944 
[144899] => 144899 
[1841306313] => Fizz 
[13973] => 13973 
[3690] => FizzBuzz 
[180817490] => Buzz 
[4811995] => Buzz 
[994975] => Buzz 
[1653] => Fizz 
[2587] => 2587 
[521] => 521 
[11878] => 11878 
[523] => 523 
[5405] => Buzz 
[166841] => 166841 
[256] => 256 
[2923] => 2923 
[78] => Fizz 
[17108] => 17108 
[19517] => 19517 
[134089] => 134089 
[62] => 62 
[4197] => Fizz 
[4334] => 4334 
[45812] => 45812 
[1448] => 1448 
[900357] => Fizz 
[334] => 334 
[61] => 61 
[18515] => Buzz 
[219] => Fizz 
[4713] => Fizz 
[8759] => 8759 
[487678485] => FizzBuzz 
[340] => Buzz 
[81823] => 81823 
[5143700] => Buzz 
[61638] => Fizz 
[23488] => 23488 
[39] => Fizz 
[4807] => 4807 
[196994] => 196994 
[29246] => 29246 
[2402] => 2402 
[50241] => Fizz 
[22242] => Fizz 
[106] => 106 
[2155] => Buzz 
[103873] => 103873 
[1329] => Fizz 
[4199093] => 4199093 
[16312] => 16312 
[1844440325] => Buzz 
[46] => 46 
[182] => 182 
[1718] => 1718 
[118751] => 118751 
[3447] => Fizz 
[1029879327] => Fizz 
[998] => 998 
[23456] => 23456 
[8610] => FizzBuzz 
[451] => 451 
[29393] => 29393 
[7742] => 7742 
[171] => Fizz 
[1510] => Buzz 
[1317368333] => 1317368333 
[449] => 449 
[5706655] => Buzz 
[714698] => 714698 
[1264] => 1264 
[635988] => Fizz 
[240740] => Buzz 
[1752322] => 1752322 
[1876] => 1876 
[15830] => Buzz 
[1736641693] => 1736641693 
[489978226] => 489978226 
[5012439] => Fizz 
[819145151] => 819145151 
[674101953] => Fizz 
[409] => 409 
[162296] => 162296 
[4179257] => 4179257 
[4024802] => 4024802 
[634] => 634 
[10300] => Buzz 
[145010] => Buzz 
[4919] => 4919 
[1303780] => Buzz 
[9430] => Buzz 
[35133] => Fizz 
[516785346] => Fizz 
[2099158545] => FizzBuzz 
[146] => 146 
[4833028] => 4833028 
[822] => Fizz 
[655809722] => 655809722 
[1308] => Fizz 
[18610] => Buzz 
[3029257] => 3029257 
[1593] => Fizz 
[630] => FizzBuzz 
[11762] => 11762 
[1789705] => Buzz 
[601505] => Buzz 
[160896] => Fizz 
[90] => FizzBuzz 
[1430614] => 1430614 
[14645] => Buzz 
[1396] => 1396 
[5712] => Fizz 
[580776111] => Fizz 
[208] => 208 
[14626] => 14626 
[215] => Buzz 
[5841372] => Fizz 
[39984] => Fizz 
[401660595] => FizzBuzz 
[5902737] => Fizz 
[70] => Buzz 
[17416] => 17416 
[86] => 86 
[119] => 119 
[1908] => Fizz 
[13556] => 13556 
[5834] => 5834 
[40004] => 40004 
[164093] => 164093 
[220] => Buzz 
[102444] => Fizz 
[115584] => Fizz 
[984768] => Fizz 
[32931] => Fizz 
[1557273573] => Fizz 
[23576] => 23576 
[41012] => 41012 
[1216930085] => Buzz 
[418312] => 418312 
[4331206] => 4331206 
[156077] => 156077 
[48] => Fizz 
[21493] => 21493 
[22086] => Fizz 
[1512] => Fizz 
[4821481] => 4821481 
[10979] => 10979 
[6007640] => Buzz 
[472282218] => Fizz 
[137617] => 137617 
[363033] => Fizz 
[7949] => 7949 
[47544] => Fizz 
[983089004] => 983089004 
[6523] => 6523 
[23988] => Fizz 
[92813] => 92813 
[5671708] => 5671708 
[12383] => 12383 
[5525993] => 5525993 
[44468] => 44468 
[1999] => 1999 
[10929] => Fizz 
[100744] => 100744 
[158320] => Buzz 
[26169] => Fizz 
[74] => 74 
[34761] => Fizz 
[493] => 493 
[223767707] => 223767707 
[5375] => Buzz 
[17060] => Buzz 
[201775] => Buzz 
[5270] => Buzz 
[16963] => 16963 
[40625] => Buzz 
[3404921] => 3404921 
[972] => Fizz 
[36991] => 36991 
[1532] => 1532 
[1551460] => Buzz 
[145321] => 145321 
[3271510] => Buzz 
[463914] => Fizz 
[3659300] => Buzz 
[166] => 166 
[797988] => Fizz 
[128] => 128 
[2678] => 2678 
[103163] => 103163 
[111] => Fizz 
[114814] => 114814 
[644] => 644 
[7320] => FizzBuzz 
[1237301528] => 1237301528 
[127] => 127 
[3371309] => 3371309 
[717] => Fizz 
[13708] => 13708 
[1638] => Fizz 
[4425706] => 4425706 
[25571] => 25571 
[167] => 167 
[5479547] => 5479547 
[980292831] => Fizz 
[2466] => Fizz 
[35913] => Fizz 
[4220] => Buzz 
[226491858] => Fizz 
[50750] => Buzz 
[129167] => 129167 
[4459] => 4459 
[1344506795] => Buzz 
[89] => 89 
[9920] => Buzz 
[10227] => Fizz 
[1757191939] => 1757191939 
[1307831] => 1307831 
[268] => 268 
[4337003] => 4337003 
[54174] => Fizz 
[49440] => FizzBuzz 
[5169] => Fizz 
[282] => Fizz 
[1997] => 1997 
[14825] => Buzz 
[500] => Buzz 
[2321080] => Buzz 
[204319999] => 204319999 
[285] => FizzBuzz 
[13016] => 13016 
[227] => 227 
[1305] => FizzBuzz 
[587] => 587 
[27342] => Fizz 
[3120] => FizzBuzz 
[28322] => 28322 
[1894017482] => 1894017482 
[545942] => 545942 
[7278] => Fizz 
[376] => 376 
[5788577] => 5788577 
[212] => 212 
[11492] => 11492 
[306] => Fizz 
[880024] => 880024 
[47588] => 47588 
[9797] => 9797 
[12826] => 12826 
[53912] => 53912 
[3611068] => 3611068 
[5711371] => 5711371 
[1232] => 1232 
[134870] => Buzz 
[1987] => 1987 
[23993] => 23993 
[772] => 772 
[28282] => 28282 
[424] => 424 
[1748901] => Fizz 
[92571] => Fizz 
[143653] => 143653 
[41591] => 41591 
[3772] => 3772 
[804611900] => Buzz 
[16227] => Fizz 
[116] => 116 
[5795] => Buzz 
[15109] => 15109 
[25048] => 25048 
[8493] => Fizz 
[37555] => Buzz 
[742] => 742 
[33320] => Buzz 
[2437] => 2437 
[514833] => Fizz 
[4040] => Buzz 
[973] => 973 
[1049] => 1049 
[63348] => Fizz 
[746] => 746 
[277] => 277 
[763418] => 763418 
[25126] => 25126 
[100651] => 100651 
[37892] => 37892 
[27384] => Fizz 
[78892] => 78892 
[560] => Buzz 
[15305] => Buzz 
[2286] => Fizz 
[1679] => 1679 
[9864] => Fizz 
[24944] => 24944 
[1494074131] => 1494074131 
[280] => Buzz 
[53848] => 53848 
[6430] => Buzz 
[5405708] => 5405708 
[840240] => FizzBuzz 
[579745] => Buzz 
[121020] => FizzBuzz 
[616] => 616 
[1688573] => 1688573 
[460190] => Buzz 
[3866410] => Buzz 
[962445456] => Fizz 
[50344] => 50344 
[5033] => 5033 
[4940] => Buzz 
[1149432734] => 1149432734 
[704138] => 704138 
[431232] => Fizz 
[807] => Fizz 
[2036] => 2036 
[5447] => 5447 
[2520550] => Buzz 
[1688445] => FizzBuzz 
[720851] => 720851 
[3444] => Fizz 
[3889] => 3889 
[32385] => FizzBuzz 
[2165499] => Fizz 
[24873] => Fizz 
[3227993] => 3227993 
[353] => 353 
[141497] => 141497 
[116409] => Fizz 
[23342] => 23342 
[4891] => 4891 
[31442] => 31442 
[16640] => Buzz 
[49443] => Fizz 
[625] => Buzz 
[4047] => Fizz 
[4126214] => 4126214 
[722776] => 722776 
[531930] => FizzBuzz 
[30182] => 30182 
[395] => Buzz 
[1304] => 1304 
[4136] => 4136 
[191] => 191 
[512] => 512 
[135] => FizzBuzz 
[2847613] => 2847613 
[3634] => 3634 
